common/src/View/FaceAttribsEditor.cpp

```diff
@@ -88,7 +88,7 @@ namespace TrenchBroom {
 
         void FaceAttribsEditor::xOffsetChanged(const double value) {
             auto document = lock(m_document);
-            if (!document->hasSelectedBrushFaces()) {
```

Comment:
This is not correct either. I think what you need is `document->hasSelectedBrushFaces() || document->selectedNodes().hasBrushes()`. Since this is used all over the place, you should introduce a new method called `hasAnySelectedBrushFaces()`; this needs to be declared in `Model::MapFacade` right next to the other corresponding methods and overridden accordingly in `MapDocument`.